File: Selection sort/selection_sort.java
class solution {
    static int[] selection_sort(int array[]) {
        int mid, temp;
        for (int i = 0; i < array.length - 1; i++) {

            mid = i;
            for (int j = i + 1; j < array.length; j++) {
                if (array[i] > array[j]) {
                    mid = j;

                    if (i != mid) {
                        temp = array[i];
                        array[i] = array[j];
                        array[j] = temp;
                    }
                }
            }
        }
        return array;
    }
}

File: selection_sort.py
class Solution: 
    def select(self, arr, i):
        # code here
        minNum = i
        for j in range(i, len(arr)):
            if arr[minNum] > arr[j]:
                minNum = j
        return minNum
    
    def selectionSort(self, arr,n):
        #code here
        for i in range(n):
            minIndex = self.select(arr, i)
            
            arr[minIndex], arr[i] = arr[i], arr[minIndex]
